src/resources/postcss/tribe-common-admin/tec_baseline_admin/_header.pcss

```
@@ -7,47 +7,37 @@
  * -----------------------------------------------------------------------------
  */
 .tec-admin-page__header {
+	align-items: center;
 	background: none;
 	background-color: transparent;
-	column-gap: var(--tec-spacer-1);
+	column-gap: var( --tec-spacer-1 );
```

```suggestion
	column-gap: var(--tec-spacer-1);
```
no spaces in `var()` or `calc()` around the parentheses.